onnx/backend/test/case/node/pool_op_common.py

```diff
@@ -95,12 +106,14 @@ def pool(
             f = np.average
         elif pooling_type == "MAX":
             f = np.max
+        elif pooling_type == "LPPOOL":
+            f = lp_pool_p
         else:
             raise NotImplementedError(
                 f"Pooling type {pooling_type} does not support. Should be AVG, MAX"
             )
 
-        if count_include_pad == 1 and pooling_type == "AVG":
+        if count_include_pad == 1 and (pooling_type == "AVG" or pooling_type == "LPPOOL"):
```

Comment:
I am curious about the existing else branch below. Does the ONNX spec specify what happens with NaN values for the pooling ops? (Not specific to current PR, but something worth checking.)

Reply:
if count_include_pad is set to 1, the caller needs to specify a value for padded pixels. In this case, there is not NaN in window_vals. If count_include_pad is not set to 0 that means the caller does not have a value for the padded pixel. In such case, padded contains NaN and so does window_vals. In this case, NaN shall be excluded from computation.
BTW compare any number with NaN is false so there is no need for MaxPool.